integrations/wiz/wiz/client.py

```diff
@@ -195,3 +209,169 @@ async def get_single_issue(self, issue_id: str) -> dict[str, Any]:
         response_data = await self.make_graphql_query(ISSUES_GQL, query_variables)
         issue = response_data.get("issues", {}).get("nodes", [])[0]
         return issue
+
+    async def setup_integration_report(self) -> None:
+        logger.info(f"Creating report '{PORT_REPORT_NAME}' to fetch issues")
+
+        report = await self.get_integration_report()
+        if report:
+            logger.info(f"Report already exists. Skipping creation: {report}")
+            return
+
+        logger.error("Report does not exist. Creating...")
```

Comment:
Do you think this should be an INFO log or an ERROR log?

Reply:
it should be info